<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="main_table">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>901</width>
     <height>201</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Нейтрофилы (NEU)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Лимфоциты (LYMF)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Общие T-лимфоциты (CD45+CD3+)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Т-хелперы (CD45+CD3+CD4+)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Т-цитотоксические лимфоциты (CD45+CD3+СD8+)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Общие В-лимфоциты (CD45+CD19+) </string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Отклонение</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Результат</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ед. Измерения</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Реф.интервал ОТ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Реф.интервал ДО</string>
    </property>
   </column>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="4" column="2">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </item>
   <item row="5" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="personal_info_table">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <row>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Имя</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Отчество</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Значение</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Дата рождения</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Дата взятия анализов</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="cytokine_table">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>891</width>
     <height>251</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>CD3+IFNy+(стимулированный)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>CD3+IFNy+(спонтанный)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>CD3+TNFa+(стимулированный)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>CD3+TNFa+(спонтанный)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>CD3+IL2+(спонтанный)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>CD3+IL4+(стимулированный)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>CD3+IL4+(спонтанный)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>CD3-IFNy+(стимулированный)</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>CD3-IFNy+(спонтанный)</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Отклонение</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Значение</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ед. Измерения</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Реф.интервал ОТ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Реф.интервал ДО</string>
    </property>
   </column>
   <item row="0" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
   <item row="4" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
   <item row="5" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
   <item row="6" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
   <item row="7" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
   <item row="8" column="2">
    <property name="text">
     <string>10Е9/л</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Цитокиновый статус</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_btn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="graph_btn">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>230</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Графики</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
